app/assets/javascripts/discourse/lib/lazy-load-images.js.es6

```diff
@@ -0,0 +1,49 @@
+const OBSERVER_OPTIONS = {
+  rootMargin: "50%" // load images slightly before they're visible
+};
+
+// We hide an image by replacing it with a transparent gif
+function hide(image) {
+  image.classList.add("d-lazyload");
+  image.classList.add("d-lazyload-hidden");
+  image.setAttribute("data-src", image.getAttribute("src"));
+  image.setAttribute(
+    "src",
+    "data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw=="
+  );
+}
+
+// Restore an image from the `data-src` attribute
+function show(image) {
+  let dataSrc = image.getAttribute("data-src");
+  if (dataSrc) {
+    image.setAttribute("src", image.getAttribute("data-src"));
```

Comment:
Any reason you're calling `getAttribute` twice?

```suggestion
    image.setAttribute("src", dataSrc);
```

Reply:
That's a mistake! Will fix, good catch.